### PR TITLE
New arrivals after 20 minutes always spawn with o2 tank and crowbar

### PR DESCRIPTION
**Pull request by Kubius**
[BALANCE][INPUT]
## About the PR 

Alters the starter emergency equipment box to change what it spawns depending on round time, guaranteeing the O2 tank spawn and adding a crowbar spawn after 20 minutes have elapsed in the round.

## Why's this needed? 

This isn't strictly necessary, but it's one possible way that changes could be made to ensure late arrivals don't end up completely screwed if arrivals is unpowered and depressurized and all the existing supplies are already exhausted.

**I am not certain that this is the correct way to go about addressing this** - input is very welcome, and I figure if this ends up not being favored, the worst that happens is I lose a patch that took less time to create than this writeup.

## Changelog

```
(u)Kubius:
(+)New arrivals after 20 minutes into the round will now always spawn with an O2 tank and crowbar.
```

### CODE REVIEW

File: code/obj/item/storage/small_storage_parent.dm [hunk 1]
```diff
@@ -299,8 +299,10 @@
 	spawn_contents = list(/obj/item/clothing/mask/breath)
 	make_my_stuff()
 		..()
-		if (prob(15))
+		if (prob(15) || ticker.round_elapsed_ticks > 20 MINUTES)
 			new /obj/item/tank/emergency_oxygen(src)
+		if (ticker.round_elapsed_ticks > 20 MINUTES)
+			new /obj/item/crowbar(src)
```
Comment (flrsh) [inline code/obj/item/storage/small_storage_parent.dm:305]:
```suggestion
			new /obj/item/crowbar/red(src)
```
This was a suggestion from the forum thread.

**Review by flrsh — APPROVED** [review]
I'm a fan.

File: code/obj/item/storage/small_storage_parent.dm (hunk 1 above)
Comment (Kubius) [reply code/obj/item/storage/small_storage_parent.dm:305]: Sensible.

**Comment by ZeWaka** [comment]
Imo, should be more like 30 or 40 minutes.

**Comment by Kubius** [comment]
30 or 40 minutes seems like a bit much. The purpose of the delay isn't just to wait until the round has gotten well and truly hosed (which _can_ actually happen at 20 minutes in when it's non-RP main) - it's more of an incentive to have people declare ready at round start and not feel bad about missing out on a guaranteed small oxygen tank and crowbar.

**Comment by flrsh** [comment]
I talked with some other admins and they thought this was fine, so I'll go ahead and merge!

**Pull request merged**